circle.yml

```diff
@@ -29,7 +29,7 @@ mainBuildFilters: &mainBuildFilters
       only:
         - develop
         - 10.0-release
-        - elevatebart/test/windows
+        - fix-cache-state
```

Comment:
Should I just drop the custom branch with this?

Reply:
It looks like @flotwig has more improvements coming down the pipeline so I'm not sure it makes a difference.